src/view/seed.js

```diff
@@ -4,16 +4,21 @@ import { observer } from 'mobx-react';
 import PropTypes from 'prop-types';
 import MainContent from '../component/main-content';
 import { CopyText, H1Text, Text } from '../component/text';
+import { Header } from '../component/header';
 import { SplitBackground } from '../component/background';
-import { GlasButton } from '../component/button';
+import { Button, BackButton, GlasButton } from '../component/button';
 import { color, font } from '../component/style';
 
 //
 // Seed View
 //
 
-const SeedView = ({ store, wallet }) => (
+const SeedView = ({ store, wallet, nav }) => (
   <SplitBackground image="purple-gradient-bg" bottom={color.blackDark}>
+    <Header>
+      <BackButton onPress={() => nav.goSelectSeed()} />
+      <Button disabled onPress={() => {}} />
+    </Header>
```

Comment:
This breaks the layout...
<img width="880" alt="bildschirmfoto 2018-09-04 um 14 03 30" src="https://user-images.githubusercontent.com/1374174/45030782-46b46d00-b04d-11e8-971b-1d1a8b1d7766.png">